libr/anal/d/cc-avr-8

```diff
@@ -0,0 +1,26 @@
+default.cc=avr
+
+avr=cc
+cc.avr.name=avr
+cc.avr.argn=r26
```

Comment:
Hmm... Argn=arg1 isn't putting arg1 into all arguments? Not understanding that either, sorry :_S